frontend/storybook/src/components/entry/imagelabel/admin/images/image-label-left.ts

```diff
@@ -0,0 +1,44 @@
+import "@elements/entry/admin/images/meta/sections/left";
+import "@elements/core/inputs/checkbox";
+import "@elements/core/inputs/text-underline";
+import "@elements/core/inputs/textarea-underline";
+import "@elements/core/buttons/text";
+import "@elements/core/dividers/horizontal-full";
+
+import {Ji as MockJiImage} from "~/components/core/images/ji";
+import "@elements/core/images/ji";
+export default {
+  title: 'Admin/Images/Settings',
+}
+
+interface Props {
+  
+}
+
+const DEFAULT_PROPS:Props = {
+}
+
+const STR_REPLACE ="Replace";
+const STR_DELETE = "Delete";
+const STR_PREMIUM ="Premium Image";
+const STR_IMAGENAME = "Image name";
+const STR_DESCRIPTION = "Image description"
+
+export const LeftLabel = (props?:Props) => {
```

Comment:
props should be `Partial<Props>` - see https://docs.sandbox.jicloud.org/book/frontend/elements-and-components/guideline-components.html#provide-arguments